install.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+if [ $# -lt 1 ]; then
+  echo "Error: missing arguments. Usage: $0 [linux|cobc] [prefix]"
+  exit 1
+fi
+
+if [[ $1 != "linux" && $1 != "cobc" ]]; then
+  echo "Error: first argument must be either 'linux' or 'cobc'"
+  exit 1
+fi
+
+if [[ $1 == "cobc" && $# -lt 2 ]]; then
+  echo "Error: missing prefix argument for 'cobc' option"
+  exit 1
+fi
+
+echo "Arguments provided: $1 $2"
+
+if [ -f /.dockerenv ]; then
+    DOCKER_BUILD=true
+else
+    DOCKER_BUILD=false
+fi
+
+PARAM_FILE="libraries.txt"
+
+while IFS=, read -r name commit repo_url; do
+  echo "Cloning $name at commit $commit from $repo_url"
```

Comment:
Rename `comit` to `reference` or `ref`, because libraries.txt contains commits and branches, and the official umbrella term for that is "reference". Also replace "commit" with "reference" in the `echo` command.